modules/wpml/load.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Loads the WPML compatibility mode.
+ *
+ * @package Polylang
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Don't access directly
+};
+
+if ( $polylang->model->get_languages_list() ) {
+	if ( ! defined( 'PLL_WPML_COMPAT' ) || PLL_WPML_COMPAT ) {
+		PLL_WPML_Compat::instance(); // WPML API
+		PLL_WPML_Config::instance(); // wpml-config.xml
```

Comment:
Why does this two have their own singletons?

Reply:
I did not change this.

Reply:
https://github.com/polylang/polylang/blob/2.7.x/include/class-polylang.php#L285-L288